added bounties (#90)
Joystream/community-repo: bounties-overview/README.md

|2  |...                          |...   |DD.MM.YY |$n    |[n](https://testnet.joystream.org/#/forum/threads/n)      |                |...  |[n](https://testnet.joystream.org/#/proposals/n)    |
|7  |Joystream Player Loading     |#85   |15.02.21 |$400  |[n](https://testnet.joystream.org/#/forum/threads/n)      |Announced       |NA   |NA                                                  |
|8  |Ledger on Joystream          |#86   |15.02.21 |$300  |[n](https://testnet.joystream.org/#/forum/threads/n)      |Announced       |NA   |NA                                                  |
|9  |Repo/Docs Improvements       |#87|15.02.21 |$400  |[n](https://testnet.joystream.org/#/forum/threads/n)      |Announced       |NA   |NA                                                  |
|10 |Upload Public Domain Content |#88   |15.02.21 |$5*  |[n](https://testnet.joystream.org/#/forum/threads/n)      |Announced       |NA   |NA                                                  |
|11 |Design Community Repo Banner |#89   |15.02.21 |$250  |[n](https://testnet.joystream.org/#/forum/threads/n)      |Announced       |NA   |NA                                                  |

## Bounties Management

> Part of the job for the Council is to manage these bounties. The tasks associated with that is:

Reviewer: ```suggestion
Part of the job for the Council is to manage these bounties. The tasks associated with that are:
```

> 1. Familiarize themselves with the tasks specified in the issue

Reviewer: ```suggestion
1. Familiarizing themselves with the tasks specified in the issue
```

> 2. Seek further information from Jsgenesis on any ambiguous or missing items

Reviewer: ```suggestion
2. Seeking further information from Jsgenesis on any ambiguous or missing items
```

> 3. Decide on the format, full workflow and process

Reviewer: ```suggestion
3. Deciding on the format, full workflow and process required
```

> 4. Create a forum thread with:

Reviewer: ```suggestion
4. Creating a forum thread with:
```

  - Link to the issue with full description
  - Specify the format and workflow

>   - If applicable

Reviewer: ```suggestion
  - If applicable:
```

    - list what it takes to be assigned the bounty

>     - assign an admin

Reviewer: ```suggestion
    - assign a dedicated manager for the bounty
```

    - set milestones/timelines
5. Update the [json](/bounties-overview/bounties-status.json) in accordance with the [schema](/bounties-overview/bounties). See explanation [here](#bounties-schema).
6. Assign a community member`*`, and update json
7. Follow up as required
8. Once a non-final spending proposal`**` is made:
  - Ensure it's in line with the workflow set
  - Review the work submitted
  - Approve if appropriate
9. Once the final spending proposal is made:

>   - Ensure it's in line with the workflow set

Reviewer: ```suggestion
  - Ensure it's in line with the workflow set out
```
